Sources/Purchasing/StoreKit2/ProductsFetcherSK2.swift

```diff
@@ -29,10 +29,17 @@ actor ProductsFetcherSK2 {
     func products(identifiers: Set<String>) async throws -> Set<SK2StoreProduct> {
         do {
             if let cachedProducts = await self.cachedProducts(withIdentifiers: identifiers) {
+                let cachedIdentifiers = Set(cachedProducts.map { $0.productIdentifier })
+                Logger.debug(Strings.offering.products_already_cached(identifiers: cachedIdentifiers))
                 return cachedProducts
             }
+            
+            Logger.debug(
+                Strings.offering.no_cached_requests_and_products_starting_skproduct_request(identifiers: identifiers)
```

Comment:
this one doesn't perfectly translate, since we don't cache requests for sk2....maybe we should?

Reply:
> maybe we should?

It's not trivial with `async`, I think that's why we don't do it.

The "no cached requests" part of the message isn't a big deal, but the latter part of the message would be weird here:
> starting `SKProducts` request for

Since that's `SK1`. So maybe it makes sense to add a new string?